enderio-base/src/main/java/com/enderio/base/common/blockentity/EIOBlockEntities.java

```diff
@@ -0,0 +1,16 @@
+package com.enderio.base.common.blockentity;
+
+import com.enderio.base.EnderIO;
+import com.enderio.base.common.block.EIOBlocks;
+import com.tterrag.registrate.Registrate;
+import com.tterrag.registrate.util.entry.TileEntityEntry;
+
+import net.minecraft.world.level.block.entity.BlockEntity;
+
+public class EIOBlockEntities {
+    private static final Registrate REGISTRATE = EnderIO.registrate();
+    
+    public static final TileEntityEntry<BlockEntity> GRAVEBE = REGISTRATE.tileEntity("gravebe", (s,p,t) -> new GraveBE(t, s, p)).validBlock(EIOBlocks.GRAVE).register();
```

Comment:
The prefix BE isn't necessary as we're in the EIOBlockEntitites class. Same thing goes for the registry name.